tempto-core/src/main/java/com/teradata/tempto/internal/fulfillment/table/TableManagerDispatcherModuleProvider.java

```diff
@@ -49,8 +53,13 @@ protected void configure()
                 Configuration databasesSectionConfiguration = configuration.getSubconfiguration("databases");
                 Set<String> databaseNames = databasesSectionConfiguration.listKeyPrefixes(1);
 
-                Map<String, ? extends Class<? extends TableManager>> tableManagerClasses = getTableManagerClassesByType();
+                if (databaseNames.isEmpty()) {
```

Comment:
without that no `newMapBinder` is called and no map is created. Guice fails that map is not created when it tries to access map of query exutoros or table managers